pkg/southbound/synchronizer/synchronizer.go

```diff
@@ -113,7 +113,7 @@ func (sync Synchronizer) syncOperationalState() error {
 	target, err := southbound.GetTarget(sync.key)
 
 	if err != nil {
-		log.Println("Can't find target for key", sync.key)
+		log.Warning("Can't find target for key", sync.key)
```

Comment:
I would be strong in using errors in this class

Reply:
done